src/cli/grain.js

```diff
@@ -67,6 +75,20 @@ const grainCommand: Command = async (args, std) => {
       recipientIdentities.size
     } identities in ${distributions.length} distributions`
   );
+  console.log();
+
+  distributions.map((distribution) => {
+    distributionMarkdownSummary(distribution, ledger, currencyDetails);
```

Comment:
distributionMarkdownSummary should call allocationMarkdownSummary directly

Reply:
If we keep them decoupled, we can do things like only log a distribution header without a breakdown of allocations.  I can abstract this collective logging that we use in `cli/grain` while keeping the underlying logging functions modular.  But let me know if you have a stronger opinion here.